apps/web/components/molecules/tabs/tabsContent.tsx

```diff
@@ -0,0 +1,23 @@
+import { Opinions } from 'molecules/opinions/opinions';
+import { Comments } from 'molecules/comments/comments';
+import { Solutions } from 'molecules/solutions/solutions';
+
+type TabObject = {
+	label: string;
+	content: () => JSX.Element;
```

Comment:
How about `ReactNode`? Does Tab component accept that?

Reply:
If I used ReactNode, in `tabs.tsx`an error was displayed
<img width="488" alt="image" src="https://user-images.githubusercontent.com/52856724/225271230-5d7e7427-face-4b7c-b90f-1d88e1223bbd.png">